.env.buildtime.schema

```
@@ -0,0 +1,15 @@
+# Build-time variables (required before Nuxt starts)
+# @defaultSensitive=false @defaultRequired=false
+# ---
+
+# Auth layer URI (build-time only, use github:DCC-BS/nuxt-layers/no-auth to disable auth)
+# @type=url
+AUTH_LAYER_URI=ifs(
+    eq($AUTH_MODE, azure),
```

dotenv-linter (4.0.0)

[warning] [IncorrectDelimiter] The eq($AUTH_MODE, azure), key has incorrect delimiter

(IncorrectDelimiter)